be/src/vec/aggregate_functions/aggregate_function_window.h

```diff
@@ -262,58 +266,31 @@ struct LeadAndLagData {
                 col.insert_default();
```

Comment:
Maybe we can sympfy this function to two case `is_null` or not.

Reply:
just for reduce one time `virtual function call`